engine/launcher/src/test/scala/org/enso/launcher/FakeEnvironment.scala

```diff
@@ -0,0 +1,33 @@
+package org.enso.launcher
+
+import java.nio.file.{Files, Path}
+import org.enso.launcher.FileSystem.PathSyntax
+
+trait FakeEnvironment { self: WithTemporaryDirectory =>
```

Comment:
I like to see testing traits documented at least, but the tests themselves don't need to be.